src/test-utils.test.ts

```diff
@@ -41,10 +41,13 @@ describe('createAPIMockingUtility', () => {
     });
 
     test('GET success with function responses', async () => {
-      network.mock('GET /items/:id', (req) => ({
-        status: 200,
-        data: { message: `${req.query.filter}|${req.params.id}` },
-      }));
+      network.mock('GET /items/:id', (req, ctx) => {
+        ctx.delay(50);
+        return {
+          status: 200,
+          data: { message: `${req.query.filter}|${req.params.id}` },
+        };
```

Comment:
Would this code actually result in a delay of `50`?

[The examples in the `msw` docs](https://mswjs.io/docs/api/context/delay#examples) don't seem to indicate that this code would work -- seems like we'd need to update the internals of our test util to support this.

It seems like we _could_ accomplish a delay using just e.g. `await wait(50)` (maybe not as easy of a solution).